pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -742,7 +743,7 @@ func (l *LibvirtDomainManager) MigrateVMI(vmi *v1.VirtualMachineInstance, option
 		return nil
 	}
 
-	if err := updateHostsFile(fmt.Sprintf("%s %s\n", "127.0.0.1", vmi.Status.MigrationState.TargetPod)); err != nil {
+	if err := updateHostsFile(fmt.Sprintf("%s %s\n", network.GetLoopbackAddress(), vmi.Status.MigrationState.TargetPod)); err != nil {
```

Comment:
I'm confused here; why aren't you re-using the `loopbackAddress` you've created previously ? 

Did I miss something ?

Reply:
here its 127.0.0.1  / ::1
there its 0.0.0.0 / ::

Reply:
I have no idea what you meant here, but I'll clarify; I mean why aren't you re-using the variable you've created in https://github.com/kubevirt/kubevirt/pull/3221/files#diff-9b1d64950edb57271a1a2fc36192d464R410 ?

AFAIU, they both refer to `network.GetLoopbackAddress()`. Don't they ?

Reply:
not same func, so var aint there